README.md

```diff
@@ -22,9 +22,9 @@
 
 ## Een introductie
 
-Na het gemeenschappelijke deel (Semester 1) van de opleiding HBO-ICT kies je een richting. Als je gekozen hebt voor Technisch Informatica (TI) dan begin je met semester 2 met als thema *Internet of Things*. Aansluitend volgen nog twee semesters in het tweede leerjaar semeter 3 *Control Technology* en semeser 4 *Complex Systems Engineering*​. In dit repository staat alle inhoud van dit Internet of Things (IoT) semester. Het project staat centraal en de inhoud zoals [hardware-interfacing](./hardware-interfacing/README.md), [software](./software/README.md) en [infrastructuur](./infrastructuur/README.md) pas je toe in het project. Er zijn [vier pijlmomenten](./Portfolio-items.md) waar we kijken naar je voortgang aan de hand van het door jouw opgestelde [verantwoordingsdocument](./Verantwoordingsdocument_TI_S2.md). Het semester 2 heeft een eigen [beroepstaak](#beroepstaak) en daarnaast zijn er negen [leeruitkomsten](#leeruitkomsten) gedefineerd deze komen gedurende dit semester stapsgewijs aan bod.
+Na het gemeenschappelijke deel (Semester 1) van de opleiding HBO-ICT kies je een richting. Als je gekozen hebt voor Technische Informatica (TI) dan begin je met semester 2 met als thema *Internet of Things* (IoT). Aansluitend volgen nog twee semesters in het tweede leerjaar: semester 3 *Control Technology* en semester 4 *Complex Systems Engineering*​. In dit repository staat alle inhoud van het *Internet of Things* semester. Het project staat centraal en de inhoud zoals [hardware-interfacing](./hardware-interfacing/README.md), [software](./software/README.md) en [infrastructuur](./infrastructuur/README.md) pas je toe in het project. Er zijn [vier pijlmomenten](./Portfolio-items.md) waar we kijken naar je voortgang aan de hand van het door jouw opgestelde [verantwoordingsdocument](./Verantwoordingsdocument_TI_S2.md). Het semester 2 heeft een eigen [beroepstaak](#beroepstaak) en daarnaast zijn er negen [leeruitkomsten](#leeruitkomsten) gedefineerd. Deze komen gedurende dit semester stapsgewijs aan bod.
 
-Onderwijs in semester 2 is projectgestuurd dat betekent dat het project centraal staat in ons onderwijs. Je werkt samen in een ontwikkelteam en je bent daarnaast verantwoordelijk voor je eigen leerprestaties. Er zijn 4 dagen in de week contactmomenten. De belasting van een voltijds studie is 42 uur per week, deels moet je dus ook thuis aan het werk. Kijk hier voor het volledige [programma van semester 2](./Programma.md)
+Onderwijs in semester 2 is projectgestuurd. Dat betekent dat het project centraal staat in ons onderwijs. Je werkt samen in een ontwikkelteam en je bent daarnaast verantwoordelijk voor je eigen leerprestaties. Er zijn 4 dagen in de week contactmomenten. De belasting van een voltijds studie is 42 uur per week, deels moet je dus ook buiten de HU (bv thuis) aan het werk. Kijk hier voor het volledige [programma van semester 2](./Programma.md)
```

Comment:
ipv 'buiten de HU' misschien beter 'buiten de lesruimte' - liefs willen we dat ze ergens aan de HU blijven of tenminste op het campus